plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesScanner.java

```diff
@@ -346,8 +331,7 @@ private void resolvePackageNames(final Report report) {
             try {
                 PackageNameResolver resolver = new PackageNameResolver();
                 resolver.run(report, getCharset());
-            }
-            catch (InvalidPathException exception) {
+            } catch (InvalidPathException exception) {
```

Comment:
It seems you reformatted the whole source with the wrong style

Reply:
its the intellij on gitpod who keeps doing it , I disabled the default settings but each time I have the same update of white spaces when I added the else block,  
do I update the formatting of else and catch to its original and I commit again ?

Reply:
Compare your changes with main and revert everything that is unrelated.